File: Source/Ivxr.SePlugin/WorldModel/MountPoint.cs
namespace Iv4xr.SePlugin.WorldModel
{
    public class MountPoint
    {
        public PlainVec3I Normal;
        public PlainVec3F Start;
        public PlainVec3F End;
        public bool Enabled;
        public bool Default;
        public bool PressurizedWhenOpen;
        public byte ExclusionMask;
        public byte PropertiesMask;
    }
}

File: JvmClient/src/commonMain/kotlin/spaceEngineers/model/MountPoint.kt
package spaceEngineers.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class MountPoint(
    @SerialName("Normal")
    val normal: Vec3F,
    @SerialName("Start")
    val start: Vec3F,
    @SerialName("End")
    val end: Vec3F,
    @SerialName("Enabled")
    val enabled: Boolean = true,
    @SerialName("Default")
    val default: Boolean = false,
    @SerialName("PressurizedWhenOpen")
    val pressurizedWhenOpen: Boolean = false,
    @SerialName("ExclusionMask")
    val exclusionMask: Byte = 0,
    @SerialName("PropertiesMask")
    val propertiesMask: Byte = 0,
)
